assets/models/system/tiiuae-falcon-40b-instruct/spec.yaml

```diff
@@ -34,4 +34,4 @@ tags:
     ]    
   inference_supported_envs:
     - vllm
-version: 3
+version: 4
```

Comment:
Please check with Sasidhar on the falcon instruct models, these are not released to the model catalog yet.